src/tasks/tames/tameTasks.ts

```diff
@@ -142,12 +142,11 @@ export async function runTameTask(activity: TameActivity, tame: Tame) {
 
 	async function handleFinish(res: { loot: Bank | null; message: string; user: MUser }) {
 		const previousTameCl = new Bank({ ...(tame.max_total_loot as ItemBank) });
-
-		const loot = res.loot?.clone() ?? new Bank();
-		const crateRes = handleCrateSpawns(user, activity.duration);
-		if (crateRes !== null) loot.add(crateRes);
+		const { loot } = res;
 
 		if (loot) {
```

Comment:
In the original code, `if (loot)` always returns true, since `const loot = .. ?? Bank()` an empty Bank will always be true. It should be `if (loot.length > 0)` however since my commit preserves the ability of loot to remain null, it is correct as-is.